packages/server/src/sdk/app/rows/search.ts

```diff
@@ -0,0 +1,38 @@
+import { Ctx, SearchFilters } from "@budibase/types"
+import { isExternalTable } from "../../../integrations/utils"
+import * as internal from "./search/internal"
+import * as external from "./search/external"
+
+export interface SearchParams {
+  tableId: string
+  paginate: boolean
+  query?: SearchFilters
+  bookmark?: number
+  limit: number
+  sort?: string
+  sortOrder?: string
+  sortType?: string
+}
+
+function pickApi(tableId: any) {
+  if (isExternalTable(tableId)) {
+    return external
+  }
+  return internal
+}
+
+export async function search(tableId: string, ctx: Ctx) {
+  return pickApi(tableId).search(ctx)
```

Comment:
I think I would prefer to keep any sort of `ctx` based functions out of the SDK if possible - these API pickers are really internal/external controllers, I would rather keep this part within the API controllers.

Reply:
Yup, this is the idea. This PR is only moving files around, the next changes are getting rid of these CTX. I wanted to split the moving code vs changing functionality into different PRs, otherwise it would be massive

Reply:
Ah I see no problem!